src/xinterpreter.cpp

```diff
@@ -42,6 +43,7 @@ void* createInterpreter(const Args &ExtraArgs = {}) {
     ClangArgs.push_back("-isystem");
     ClangArgs.push_back(CxxInclude.c_str());
   }
+#endif
```

Comment:
These instructions( `DetectResourceDir` and `DetectSystemCompilerIncludePaths `)  can never be put to use as they are based on `exec` which emscripten doesn't plan on supporting now or in future.

Reply:
**Even if we are okay having errors coming up in the console due to this. They lead to other problems anyways.**

![image](https://github.com/user-attachments/assets/1588d6a7-2fbc-4e6f-b0df-4fe9a5c654c7)